.changeset/config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@2.3.1/schema.json",
+  "changelog": [
+    "@changesets/changelog-github",
+    { "repo": "single-spa/single-spa" }
+  ],
+  "commit": false,
+  "fixed": [],
+  "linked": [],
+  "access": "restricted",
+  "baseBranch": "6.0",
```

Comment:
My plan is to rename the `6.0` branch to `main`, and rename `master` to `5.x`, once we publish single-spa@6 as latest. Once that happens, we'll need to update the changeset baseBranch to `main`